app/src/main/java/fr/free/nrw/commons/explore/images/SearchImagesRenderer.java

```diff
@@ -63,10 +63,10 @@ public void render() {
      * formats author name as "Uploaded by: authorName" and sets it in textview
      */
     private void setAuthorView(Media item, TextView author) {
```

Comment:
Author again now seems like the wrong name here

Reply:
yeah will do that. requires modifying textview as well, author is used everywhere.
`@BindView(R.id.categoryImageAuthor) TextView categoryImageAuthor;`